assets/app/scripts/filters/util.js

```diff
@@ -105,6 +105,12 @@ angular.module('openshiftConsole')
         // We didn't get an amount? shouldn't happen but just in case
         return [value, null];
       }
+
+      // No normalization of unit.
+      if (!humanizeUnits) {
+        return split.slice(1);
```

Comment:
So this is stripping the unit and only returning the value?  I think I would rather have usageValue take a flag that prevents normalization.  It's weird that the default behavior of a function called amountAndUnit is to drop the unit.